app/views/sp_components/_all_sp_components.html.erb

```diff
@@ -15,7 +16,11 @@
           <td class="govuk-table__cell">
             <%= component.view_component_type(component.vsp) %>
           </td>
-          <td class="govuk-table__cell"><%= link_to t('components.view_link'), sp_component_path(component) %></td>
+          <td class="govuk-table__cell" scope="row"><%= component&.team&.name %></td>
```

Comment:
```suggestion
          <td class="govuk-table__cell" scope="row"><%= component.dig(:team, :name) %></td>
```
The component does exist, no need to check with &... but we can use dig instead for the team name

Reply:
thanks